module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -49,6 +49,8 @@ abstract class MinkTestCase extends \PHPUnit\Framework\TestCase
     use \VuFindTest\Feature\AutoRetryTrait;
     use \VuFindTest\Feature\LiveDetectionTrait;
 
+    public const DEFAULT_TIMEOUT = 5000;
```

Comment:
A general thought on the centralization/standardization of timeout values: we currently have a SNOOZE_MULTIPLIER environment variable for slowing things down in busy/laggy environments. Should that variable increase timeout lengths in addition to extending snoozes? I imagine it may be helpful to have a mechanism for extending timeouts. I suppose the real test will be what happens when we run this in the live Jenkins environment, and I'm not advocating for over-engineering everything before there's a need... but just something to think about!

Reply:
Potentially yes, but my aim is to make it so that extending snooze is never needed. I think it might be clearer to allow overriding the timeout rather than multiplying it.

Reply:
Latest commit eliminates snooze from everywhere but the retries in `setValue` and `clickCss`. I haven't seen them fail during testing, however, so I'm not sure if they are really needed, but there's no harm in keeping them for now.

I also made the default timeout configurable by setting VUFIND_DEFAULT_TIMEOUT environment variable (milliseconds).